### DISCUSSION TITLE
How to use windowrulev2 across multiple workspaces

### DISCUSSION THREAD

**Discussion by Kommynct** [hyprwm/Hyprland#5381] [Q&A] [answered]
I tried

windowrulev2 = bordersize 0, workspace:r[1-10],class:(firefox)

as a workaround for my problem https://github.com/hyprwm/Hyprland/issues/5325#issuecomment-2028520233

do I have to manually remake that rule for 1-10?

**Comment by vaxerski** [comment, marked as answer]
you mean `onworkspace:`?

**Reply by Kommynct** [reply]
It is extremely unclear to me what the difference is, I don't know why both should exist at all, but this does resolve the issue, the format is 

windowrulev2 = bordersize 0, onworkspace:r[1-10],class:(firefox)

**Reply by vaxerski** [reply]
they shouldn't. It's a remnant.
